src/components/dev-hub/community-champion-application.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import { fontSize, screenSize, size } from './theme';
+import { H5, P } from './text';
+import Button from './button';
+import CommunityChampionForm from './community-champion-form';
+import Modal from './modal';
+import SuccessState from './success-state';
+
+const CONTENT_STYLE_MOBILE_BOTTOM_PADDING = '48px';
+const CONTENT_STYLE_BOTTOM_PADDING = '40px';
+const TITLE_MOBILE_LINE_HEIGHT = '30px';
+const DESCRIPTION_BOTTOM_MARGIN = '40px';
+const SUCCESS_STATE_BOTTOM_MARGIN = '40px';
+const SUCCESS_STATE_HEADING_MOBILE_FONT_SIZE = '28px';
+const SUCCESS_STATE_HEADING_MOBILE_LINE_HEIGHT = '38px';
+const MODAL_DIALOG_CONTAINER_STYLE_WIDTH = '600px';
+
+const ModalContainer = styled('div')`
+    @media ${screenSize.mediumAndUp} {
+        padding: 0 ${size.mediumLarge};
+    }
+`;
+
+const contentStyle = css`
+    @media ${screenSize.upToMedium} {
+        border: none;
+        border-radius: 0;
+        max-width: unset;
```

Comment:
I feel like this line, combined with some of the others here, might not actually be needed (I commented out the max-width, min-height, min-width, and width and they seemed to not make an impact?) Do you see that as well?

Reply:
hmm yeah, i don't think max-width, min-width and width are needed here but min-height is needed for the form to go full-height. you should be able to see a difference in the success state modal. without min-height set, height will just expand to fit the content inside